specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json

```diff
@@ -495,6 +495,75 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.PolicyInsights/policyStates/latest/triggerEvaluation": {
+      "post": {
+        "operationId": "PolicyStates_TriggerSubscriptionEvaluation",
+        "description": "Triggers a policy evaluation scan for all the resources under the subscription",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The scan was successfully triggered.",
+            "schema": {
+              "$ref": "#/definitions/TriggerEvaluationResults"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/TriggerEvaluationFailure"
+            }
+          }
+        },
+        "x-ms-examples":{
+          "Trigger evaluations for all resources in a subscription": {
+            "$ref": "./examples/PolicyStates_TriggerSubscriptionEvaluation.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PolicyInsights/policyStates/latest/triggerEvaluation": {
+      "post": {
+        "operationId": "PolicyStates_TriggerResourceGroupEvaluation",
+        "description": "Triggers a policy evaluation scan for all the resources under the subscription",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The scan was successfully triggered.",
+            "schema": {
+              "$ref": "#/definitions/TriggerEvaluationResults"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/TriggerEvaluationFailure"
```

Comment:
>TriggerEvaluationFailure [](start = 37, length = 24)

Please re-use the existing error definition for this spec "#/definitions/QueryFailure"

Reply:
Done